README.md

```diff
@@ -34,14 +34,15 @@ GUARD Tool
 Usage: guard [OPTIONS]
 
 Options:
-  -h,--help              Use the below listed functions.
-                         Warning: Don't try guard on non guardable units
-                         (sys, perv)
-  -c,--create TEXT       Create GUARD record, expects physical path as input.
-  -d,--delete TEXT       Delete GUARD record, expects physical path as input.
-  -l,--list              Listing of GUARDed resources.
-  -r,--clearall          Clears GUARD states for all resources.
-  -v,--version           Version of GUARD tool.
+  -h,--help                Guard CLI tool options
+  -c,--create TEXT         Create Guard record, expects physical path as input
+  -i,--invalidate TEXT     Invalidate a single Guard record, expects physical
+                           path as input
+  -I,--invalidate-all      Invalidates all the Guard records
+  -l,--list                List all the GUARD'ed resources
+  -r,--reset               Erase all the Guard records
+  -v,--version             Version of GUARD tool
+
```

Comment:
- a  details option is missing here.

Reply:
Not added -a details as it is hidden in guard tool.
In readme only adding the options shown in help text(i.e. guard -h)